.github/workflows/rust.yml

```
@@ -36,7 +36,7 @@ jobs:
     - run: rustup update
     - run: rustup +nightly component add rust-analyzer
     - name: Check if rust-analyzer encounters any errors parsing project
-      run: rustup run nightly rust-analyzer analysis-stats . 2>&1 | (! grep ERROR)
+      run: rustup run nightly rust-analyzer analysis-stats . 2>&1 >/dev/null | (! grep ERROR)
```

We just fixed this in another repo with the combination of these changes:
- https://github.com/stellar/rs-soroban-sdk/pull/994
- https://github.com/stellar/rs-soroban-sdk/pull/995

The output is important, we do not want to send it to dev null. Instead we need to improve the match:

```suggestion
      run: rustup run nightly rust-analyzer analysis-stats . 2>&1 | (! grep '^\[ERROR')
```

---

I wasn't aware of that fix on the other repos. Let me update it here.